### PR TITLE
refactor: remove store currency

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### Things to Talk About
Plz check by commits

### CODE REVIEW

**Review by WANZARGEN — APPROVED** [review]
LGTM~~~

**Review by yuda110 — APPROVED** [review]
LGTM!!!

File: apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisHeader.vue [hunk 1]
```diff
@@ -104,7 +103,7 @@ const handleDeleteQueryConfirm = async () => {
                     </div>
                     <div class="title-right-extra currency-wrapper">
                         <span class="label">{{ $t('BILLING.COST_MANAGEMENT.COST_ANALYSIS.CURRENCY') }}:</span>
-                        <span>{{ state.currencySymbol }}{{ state.currency }}</span>
+                        <span>{{ CURRENCY_SYMBOL.USD }}{{ CURRENCY.USD }}</span>
```
Comment (sulmoJ) [inline apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisHeader.vue:106]: Issue: This `currency` information should be based on the currency that the dataSource has.

**Review by sulmoJ — APPROVED** [review]
LGTM! 
I left a Issue!

**Pull request merged**